README.md

```diff
@@ -49,7 +49,7 @@ distributions. Please ensure that your cluster distribution supports these requi
 
 [reachable from kube-apiserver]: https://kubernetes.io/docs/concepts/architecture/master-node-communication/#master-to-cluster
 [enable an aggregation layer]: https://kubernetes.io/docs/tasks/access-kubernetes-api/configure-aggregation-layer/
-[authentication and authorization]: https://kubernetes.io/docs/reference/command-line-tools-reference/kubelet-authentication-authorization/
+[authentication and authorization]: https://kubernetes.io/docs/reference/access-authn-authz/kubelet-authn-authz/
```

Comment:
thanks for the fix.
I think the L148-L150 have also failed, could you repaired then together?
https://github.com/kubernetes-sigs/metrics-server/blob/115e821ecd72aa1faac0b9257e9c488650cac8f4/README.md?plain=1#L148

Reply:
I fixed.
As [the document](https://github.com/kubernetes/community/tree/master/contributors/design-proposals) said, the design proposal is already archived. So I replaced links with the archive repository.